docs_user/modules/proc_adopting-the-block-storage-service.adoc

```diff
@@ -74,6 +74,61 @@ spec:
         customServiceConfig: |
           [DEFAULT]
           default_volume_type=tripleo
+      cinderScheduler:
+        replicas: 0
+      cinderBackup:
+        networkAttachments:
+        - storage
+        replicas: 0
+      cinderVolumes:
+        ceph:
+          networkAttachments:
+          - storage
+          replicas: 0
+----
+
+. Retrieve the list of the previous scheduler and backup services:
++
+----
+$ openstack volume service list
+
++------------------+------------------------+------+---------+-------+----------------------------+
+| Binary           | Host                   | Zone | Status  | State | Updated At                 |
++------------------+------------------------+------+---------+-------+----------------------------+
+| cinder-scheduler | standalone.localdomain | nova | enabled | down  | 2024-11-04T17:47:14.000000 |
+| cinder-backup    | standalone.localdomain | nova | enabled | down  | 2024-11-04T17:47:14.000000 |
+| cinder-volume    | hostgroup@tripleo_ceph | nova | enabled | down  | 2024-11-04T17:47:14.000000 |
```

Comment:
adoption code cleans up only scheduler and backup services which are down but not volume, shouldn't we cleanup this entry in db ?

Reply:
We should but what happens is, when we create the new cinder-volume service in RHOSO, we use the same hostname which re-uses the service record for cinder-volume.
In the documentation, we mention to clean up all the down services but in the ansible test job, we don't need to do it.
We can see that in effect when the job runs successfully.

Reply:
Thanks for the clarification.